content/docs/overview/client-integrations/index.md

```diff
@@ -25,21 +25,21 @@ libraries from GitHub's CDN and initialize by populating the `observerWsEndPoint
   - audio bridge plugin
 
 It should support other plugins as well out of the box. But, if you find some issue with it and if it does not fully support other plugins (s), please create an issue or pull request. Thanks
-    
+
 See the quickstart methodology below for adding this library to your web app.
 
-### Janus Quickstart <a name="quickstart"></a>
+### Janus Quickstart
```

Comment:
The source of the client-integration is the wiki page like https://github.com/ObserveRTC/integrations/wiki/Vonage-OpenTok-Integration I need to change the script fetching the integration repository and bundle together the integrations into one page.